<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueItem</class>
 <widget class="QMainWindow" name="IssueItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check Out</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>65</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>UserID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userID">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="userID_search_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>491</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>34</number>
     </property>
     <property name="verticalSpacing">
      <number>17</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="UNameLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Not Available</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="UUNameLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Not Available</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>65</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>ItemID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="itemID">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>501</width>
      <height>167</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <property name="horizontalSpacing">
      <number>31</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ExtraLabel">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Availability:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="INameLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Not Available</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="IExtraLabel">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Not Available</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="IAvailabilityLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
       </property>
       <property name="text">
        <string>Not Available</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="itemID_search_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="issue_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Issue Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
